packages/salesforcedx-vscode-core/test/jest/context/workspaceContext.test.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { WorkspaceContextUtil } from '@salesforce/salesforcedx-utils-vscode';
+import { WorkspaceContext, workspaceContextUtils } from '../../../src/context';
+import * as contextVariables from '../../../src/context/contextVariables';
+import { decorators } from '../../../src/decorators';
+
+describe('workspaceContext', () => {
+  let setupWorkspaceOrgTypeMock: jest.SpyInstance;
+  let workspaceContextUtilGetInstanceSpy: jest.SpyInstance;
+  let setIsScratchOrgSpy: jest.SpyInstance;
```

Comment:
Since setIsScratchOrgSpy and decoratorsMock aren't created in this describe/beforeEach,  to scope them they should probably be moved to line 35.

Also, as I mentioned earlier, `clearAllMocks()` should be called in `afterEach()`, but with these split like they are, I'm not sure where the best place to call `clearAllMocks()` would be.